2.4/Dockerfile.fedora

```diff
@@ -0,0 +1,79 @@
+# FROM registry.fedoraproject.org/f26/s2i-base:latest
+FROM f27/s2i-core
+
+# This image provides a Ruby 2.4 environment you can use to run your Ruby
+# applications.
+
+EXPOSE 8080
+
+ENV NAME=ruby \
+    RUBY_VERSION=2.4 \
+    VERSION=0 \
+    RELEASE=1 \
+    ARCH=x86_64
+
+ENV SUMMARY="Platform for building and running Ruby $RUBY_VERSION applications" \
+    DESCRIPTION="Ruby $RUBY_VERSION available as docker container is a base platform for \
+building and running various Ruby $RUBY_VERSION applications and frameworks. \
+Ruby is the interpreted scripting language for quick and easy object-oriented programming. \
+It has many features to process text files and to do system management tasks (as in Perl). \
+It is simple, straight-forward, and extensible."
+
+LABEL summary="$SUMMARY" \
+      name="$FGC/$NAME" \
```

Comment:
Please forgive my ignorance- where does the $FCG come from?
(Also, why is there a different var in [postgresql](https://github.com/sclorg/postgresql-container/blob/master/latest/Dockerfile.fedora#L36)?)

Reply:
lol, it's a typo in postgres dockerfle: https://github.com/sclorg/postgresql-container/pull/223

FGC is a variable defined in the fedora base image:

```
$ docker run registry.fedoraproject.org/fedora:26 env | grep FGC
FGC=f26
```

https://fedoraproject.org/wiki/Container:Guidelines#Fedora_Base_Image

Reply:
Great, thanks!